example/feature_demo/demo_service.pb.gorm.go

```diff
@@ -17,8 +17,6 @@ import google_protobuf2 "github.com/golang/protobuf/ptypes/empty"
 var _ = fmt.Errorf
 var _ = math.Inf
 
-// IntPointORM is a basic message type representing a single cartesian point
-// that we want to store in a database
```

Comment:
I notice the comment is no longer propagated from the proto file to the gorm output. Is this something that is useful enough to include, or is dropping it fine? Thoughts @dmacthedestroyer

Reply:
Yeah, I accidentally threw away them while refactoring the code. 
I think anyway we miss comments propagation for services and methods, probably something else, fields for example.
So we can create a separate story/task for this, probably this can be handled in a unified way across different proto entities.

Reply:
I'm fine with that. Those comments are available in the proto file, having them in the gorm file as well is not so critical and can be done at a later time.